bundle-workflow/jenkins_workflow/test/Jenkinsfile

```diff
@@ -34,6 +34,11 @@ pipeline {
                                             name: 'test_run_id',
                                             trim: true
                                     ),
+                                    string(
+                                            defaultValue: '',
+                                            name: 'test_suite_type',
```

Comment:
Nit, `test_suite_type` -> `test_suite`, I think it's equally clear but shorter.

Reply:
Sure, I like it. I have a follow up PR, will clean it up there.